ui/src/common/QualityInfo.js

```diff
@@ -5,7 +5,8 @@ export const QualityInfo = (props) => {
   let { suffix, bitRate } = props.record
   suffix = suffix.toUpperCase()
   let info = suffix
-  if (bitRate !== '') {
+  let lossLessFormats = ['FLAC', 'WAV', 'DSF']
```

Comment:
Can you move this list to the `consts.js` file? And use a uppercase  name. Like this:

```js
const LOSSLESS_FORMATS = ['flac', 'wav', 'dsf']
```

(the values have to be lowercase because of the comparison below)

Reply:
Will move the list but as i have converted the suffix to uppercase before comparison Ii don't think that we need lowercase values in the list.